shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/helpers/JsonApiExceptionError.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.blisskmmic.data.network.helpers
+
+import co.nimblehq.jsonapi.model.JsonApiException
+
+fun Throwable.jsonApiException() : String? {
+    when(this) {
+        is JsonApiException -> return this.errors.first().detail
```

Comment:
Do we need to handle the else case here? Shouldn't there be a warning about this? 🤔

Reply:
I will move the else block inside the when block for clarity.